elementary/utils/log.py

```diff
@@ -25,6 +26,8 @@ def format(self, record):
 
 
 FORMATTER = ColoredFormatter()
+MAX_BYTES_IN_FILE = 10240000
```

Comment:
Pretty arbitrary choice from some examples I saw. No hard feelings if you want me to change that

Reply:
Maybe we can [copy dbt](https://github.com/dbt-labs/dbt-core/blob/9bb1250869d6d004f2a81656986bd03c03cf12da/core/dbt/logger.py#L357).

Reply:
It's pretty much the same (10240000  is also around 10 mb). But theirs is more readable so I'll change it :)